Formula/podman.rb

```diff
@@ -17,6 +17,25 @@ class Podman < Formula
   depends_on "go-md2man" => :build
   depends_on "qemu" if Hardware::CPU.intel?
 
+  on_linux do
+    depends_on "conmon"
+    depends_on "crun"
+    depends_on "gpgme"
+    depends_on "libseccomp"
+  end
+
+  on_linux do
+    resource "cni-plugins" do
+      url "https://github.com/containernetworking/plugins/archive/v1.0.0.tar.gz"
+      sha256 "8331f2c09d13012baacdec4d0ce07c081d184354a6c4e5aba40584a88a53437f"
+    end
+
+    resource "containers.conf" do
```

Comment:
We need this to override the default `cni_plugin_dirs` so `podman` can find our plugins. I was hoping we could install our own so there's no manual configuration needed, but the only filepaths searched are https://github.com/containers/common/blob/6143ffe54e5f9da2f69a4d231c1123af07fdc1e4/pkg/config/containers.conf#L11-L13 which i don't think we can write to?

Reply:
Usually patch it in the source code (so that no configuration is needed), or add symlinks to the default locations.